packages/core/src/queries/users-roles.ts

```diff
@@ -1,28 +1,40 @@
 import type { CreateUsersRole, UsersRole } from '@logto/schemas';
-import { RolesScopes, UsersRoles } from '@logto/schemas';
+import { UsersRoles, Roles, RoleType } from '@logto/schemas';
 import { conditionalSql, convertToIdentifiers } from '@logto/shared';
 import type { CommonQueryMethods } from 'slonik';
 import { sql } from 'slonik';
 
 import { DeletionError } from '#src/errors/SlonikError/index.js';
 
-const { table, fields } = convertToIdentifiers(UsersRoles);
+const { table, fields } = convertToIdentifiers(UsersRoles, true);
+const { fields: insertFields } = convertToIdentifiers(UsersRoles);
 
 export const createUsersRolesQueries = (pool: CommonQueryMethods) => {
   const countUsersRolesByRoleId = async (roleId: string) =>
     pool.one<{ count: number }>(sql`
       select count(*)
       from ${table}
-      where ${fields.roleId}=${roleId}
+      join ${sql.identifier([Roles.table])} on ${sql.identifier([
```

Comment:
The table `UsersRoles` is expected to have only "user" roles, this should be garranteed by an alteration script?